internal/cephfs/core/volume.go

```diff
@@ -190,7 +196,12 @@ func (s *subVolumeClient) GetSubVolumeInfo(ctx context.Context) (*Subvolume, err
 	return &subvol, nil
 }
 
-type operationState int64
+type operationState int32
+
+type operationStateMutex struct {
```

Comment:
This seems to be introduced to prevent concurrent access to the int32. Are there really issues with reading/writing the values concurrently?

Reply:
> This seems to be introduced to prevent concurrent access to the int32. Are there really issues with reading/writing the values concurrently?

None have been seen, but being precautionary here.

Reply:
I don't see a problem with normal int usage. If there is no explanation of why it could be problematic, don't add the extra (unneeded) complexity.